docs/jwst/resample/arguments.rst

```diff
@@ -15,35 +15,72 @@ image.
     image.  Available kernels are `square`, `gaussian`, `point`,
     `turbo`, `lanczos2`, and `lanczos3`.
 
+    For spectral data, only the `square` and `point` kernels should be used.
+
 ``--pixel_scale_ratio`` (float, default=1.0)
-    Ratio of input to output pixel scale.  A value of 0.5 means the output
+    Ratio of input to output pixel scale.
+
+    For imaging data, a value of 0.5 means the output
     image would have 4 pixels sampling each input pixel.
+
+    For spectral data, values greater than 1 indicate that the input
+    pixels have a larger spatial scale, so more output pixels will
+    sample the same input pixel.  For example, a value of 2.0
+    means the output image would have 2 pixels sampling each input
+    spatial pixel. If the input data has units of flux density (MJy/pixel),
+    the output flux per pixel will be approximately half the input flux
```

Comment:
With real data, it depends a bit on how the input/output pixel areas overlap - flux in one input pixel might be spread across several output pixels. But if the data were perfectly uniform, it would really be half, so that part is not approximate.  I'll take out the 'approximately' in this sentence.